packages/app-extensions/src/appFactory/appFactory.js

```diff
@@ -29,7 +29,8 @@ export const createApp = (
     const initIntlPromise = setupIntl(input, store, name, textResourceModules)
 
     return {
-      component: <App store={store} initIntlPromise={initIntlPromise} content={content} theme={theme} />,
+      component: <App store={store} initIntlPromise={initIntlPromise} content={content} theme={theme}
+                      themeType={input.themeType === 'WIDGET' ? 'WIDGET' : 'ADMIN'}/>,
```

Comment:
tiny and picky comment:
```
input.themeType || 'ADMIN' 
```
For me, with the `||` logic its's more clear that `'ADMIN'` is the "fallback" or "default". Also, when we would have more types later this code would still be valid.